frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -223,15 +256,12 @@ export const VMSchedulingList: React.FC<VMSchedulingListProps> = ({
   vmi,
   canUpdateVM,
   kindObj,
+  openFlavorModal = false,
+  setOpenFlavorModal,
 }) => {
-  const isVM = kindObj === VirtualMachineModel;
-  const vmiLike = isVM ? vm : vmi;
+  const vmiLike = !isVMRunningOrExpectedRunning(vm) ? vm : vmi;
```

Comment:
this will display wrong information about VM's, once you change for example flavor

Reply:
The feature is about pending changes for the next run.
If a vm is running you want to show the current configuration while the pending changes you can see it the modal - this way you can see the difference between what you have now and what will you have after restart.
It make much more sense to me.

Reply:
Why does it make more sense to show the VMI configuration? and not VM configuration?

I think it is a bit weird from user perspective.

\- you click on flavor
\- change flavor in modal
\- the flavor you clicked on is not changed 
\- you will start asking if I clicked cancel or save?

\- also the user might expect the VM details to respect the VM yaml
\- you have to click modal to see the real values
\- if you are interested in the VMI view - you can find a VMI page

Anyway this change will also cause inconsistencies.
\- why is it okay to show VMI cdroms and not disks?
\- why do we show VMI info of scheduling part, but keeping the os for VM? plus you cannot believe the overview of scheduling modal will have something different inside

IMO, this will cause that the users will be unsure what is real info on VM's page and what is not.
Can we just show a link to VMI and describe which fields are not included in VMI configuration?